packages/analytics-types/src/config.ts

```diff
@@ -60,23 +60,19 @@ export type InitOptions<T extends Config> =
       };
 
 export interface TrackingOptions {
-  city?: boolean;
-  country?: boolean;
-  carrier?: boolean;
   deviceManufacturer?: boolean;
   deviceModel?: boolean;
-  dma?: boolean;
   ipAddress?: boolean;
   language?: boolean;
   osName?: boolean;
   osVersion?: boolean;
   platform?: boolean;
-  region?: boolean;
-  versionName?: boolean;
+  [key: string]: boolean | undefined;
```

Comment:
I added this to make the changes here backwards compatible for the removed properties

Reply:
We don't track these in react native?

Reply:
@qingzhuozhen region and app version being tracked but not controlled by tracking options as of the moment